core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala

```diff
@@ -88,6 +89,7 @@ object MasterToAppMaster {
   case class AppMasterDataDetail(appId: Int, appDescription: Application)
   case class ReplayFromTimestampWindowTrailingEdge(appId: Int)
   case class GetAppDataResult(key: String, value: Any)
+  case class WorkerList(workers: List[Int])
   case object AppDataReceived
 }
 
```

Comment:
@kkasravi 

Yeah, you are right. We should do that. 

Before that, we want to add more experiment projects, so that we can abstract out a more general AppMaster.